ledger.go

```diff
@@ -457,6 +472,23 @@ func (l *Ledger) FinalizeRounds() {
 
 FINALIZE_ROUNDS:
 	for {
+		// Add sender transactions waiting in the queue
+		txAvailable := true
+		for txAvailable {
+			select {
+			case tx := <-l.txQueue:
+				tx = AttachSenderToTransaction(
+					l.client.Keys(), tx, l.graph.FindEligibleParents()...)
+
+				if err := l.addTransaction(tx); err != nil {
+					fmt.Println("failed to add queued transaction")
+				}
+
+			default:
+				txAvailable = false
```

Comment:
Can just break the for loop here rather than use a `txAvailable` bool.

Reply:
Really? `go vet` is saying that the code after that is unreachable. I think using `break` will break out of the `case` statement, not the `for` loop.